.github/workflows/docker.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+    push:
+        branches: [ master ]
+    release:
+        types: [published]
+
+jobs:
+    docker_build:
+        name: Create and push Docker image to Docker Hub
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check out the repo
+              uses: actions/checkout@v2
+            - name: Set up JDK 11
+              uses: actions/setup-java@v1
+              with:
+                  java-version: 11
+            - name: Build with Maven
+              run: mvn -B package --file pom.xml
+            - name: Push to Docker Hub
+              uses: docker/build-push-action@v1
+              with:
+                  username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
[non-blocking]:
Is this really necessary?

Reply:
Yes - I need the updated Docker image on Docker hub in order to recognize it on the CI server and update the container. The only other option would be to have a private Docker registry which we can use. At the moment I do not see that this is necessary.